Source/Core/Common/Network.cpp

```diff
@@ -170,6 +182,50 @@ u8 UDPHeader::IPProto() const
   return static_cast<u8>(IPPROTO_UDP);
 }
 
+ARPHeader::ARPHeader() = default;
+
+ARPHeader::ARPHeader(u32 from_ip, MACAddress from_mac, u32 to_ip, MACAddress to_mac)
+{
+  hardware_type = 0x100;
+  protocol_type = 8;
```

Comment:
some of these constants are probably `define`d in some network header that's already included?